examples/flava/definitions.py

```diff
@@ -64,7 +64,9 @@ class TrainingDatasetsInfo:
 @dataclass
 class TrainingArguments:
     # Any lightning args to be pushed here
-    lightning: Dict[str, Any] = field(default=dict)
+    lightning_trainer: Dict[str, Any] = field(default=dict)
```

Comment:
another way would be to create a data class wrapping around trainer conf and checkpoint conf. but upto u if u want to do it or just leave it this way

Reply:
Yes, we can extract all lightning config params to standalone Lightning config dataclass